### PR TITLE
12 flag projects

### LINKED ISSUE #12: User can flag projects for data issues
For example, if a link to CPC report is missing

### PR DESCRIPTION
**Pull request by chriswhong**
Adds a pop-up feedback form to the project view.  
The form includes a recaptcha, and sends the project name, project id, and user's feedback to the backend.

Depends on [this PR](https://github.com/NYCPlanning/labs-zap-api/pull/33) for the API changes.

Closes #12

### CODE REVIEW

File: app/controllers/show-project.js [hunk 1]
```diff
@@ -10,6 +10,11 @@ export default class ShowProjectController extends Controller {
   @argument shareClosed = true;
   @argument copySuccess = false;
 
+  flagText = '';
+  flagClosed = true;
+  flagSuccess = false;
+  reCaptchaResponse = null;
```
Comment (allthesignals) [inline app/controllers/show-project.js]: In general, unless these things depend on query params (which are inherently Controller level properties), stuff like this should be kept inside a component, including the `bblFeatureCollectionLayer`.

File: app/controllers/show-project.js [hunk 2]
```diff
@@ -72,4 +77,52 @@ export default class ShowProjectController extends Controller {
...
+  }
+
+  @action
+  submitFlag() {
+    const projectname = this.get('model.dcp_projectname');
+    const projectid = this.get('model.dcp_name');
+    const flagText = this.get('flagText');
+    const reCaptchaResponse = this.get('reCaptchaResponse');
+
+    fetch(`http://localhost:3000/projects/feedback`, {
```
Comment (allthesignals) [inline app/controllers/show-project.js]: This won't work in production... we should use environment variables for this.

File: app/controllers/show-project.js (hunk 1 above)
Comment (chriswhong) [reply app/controllers/show-project.js]: Done

File: app/controllers/show-project.js (hunk 2 above)
Comment (chriswhong) [reply app/controllers/show-project.js]: Done

**Comment by allthesignals** [comment]
Thanks! 🙌

**Pull request merged**